### ISSUE TITLE
Making the bot reply with the request info comment after manually adding a label manually

### ISSUE THREAD

**Issue by jahirfiquitiva** [behaviorbot/request-info#30]
Hi...
First of all, thanks for this awesome bot.

I was wondering if you could add an option to make the bot reply with the request info comment after one adds a label manually.

Similar to what the [support bot](https://probot.github.io/apps/support/) does after one adds the "support" (or a custom) label.

I think it would be useful as some people sometimes put some content in the issue but it isn't still enough and more info should be added, or when people does not fill an issue template properly.

Thanks in advance. :)

**Comment by hiimbex** [comment]
So it would comment when a label, for example, `request-info` info is added?

that seems like a good idea to me! any interest in implementing this?

**Comment by jahirfiquitiva** [comment]
@hiimbex 
Yes, basically. It could be a separate config so it doesn't conflict with the label the bot adds when commenting. Maybe in this case the bot would not add the label if it's already added (manually).

About interest in implementing this, I totally would but I honestly don't have the skills to do so :sweat:

**Comment by knrt10** [comment]
This seems pretty nice let me try to implement this one.

**Comment by murrant** [comment]
This would be very helpful.  It would be even more helpful if you could map labels to different responses.

That way you could have canned responses for different types of info requests.
